src/xml2_namespace.cpp

```diff
@@ -7,8 +9,8 @@
 #include "xml2_types.h"
 #include "xml2_utils.h"
 
-// [[export]]
-extern "C" SEXP unique_ns(SEXP ns) {
+[[cpp11::register]]
+cpp11::sexp unique_ns(SEXP ns) {
   BEGIN_CPP
```

Comment:
Remove?

Reply:
And in the other places below. Do we need this macro at all anymore? I assume if you go through cpp11 everywhere it will wrap it in try/catch for you

Reply:
@DavisVaughan If you could merge #400 then I could also finish xml2_node.cpp and I think then it should be safe to remove the macros.